packages/forklift-console-plugin/src/modules/Providers/utils/validators/secret-fields/vsphereSecretFieldValidator.ts

```diff
@@ -11,6 +11,7 @@ import { validateFingerprint, validateNoSpaces } from '../common';
  * 'default' - The default state of the form field, used when the field is empty or a value hasn't been entered yet.
  * 'success' - The field's value has passed validation.
  * 'error' - The field's value has failed validation.
+ * 'warning' - The field's value might fail the validation, but it's not mandatory and not disabling the form saving.
  */
```

Comment:
...but the field is not mandatory and if it fails, the form saving will not be disabled."

Reply:
Correct. It's just a comment for explaining the warning validation state.
In case of a warning validation result for a field - a message will appear but the form saving will not be disabled.

Reply:
> The field's value might fail the validation, but it's not mandatory and not disabling the form saving.

the reason the "create" is not disabled is because the value is valid and will be used as it is.
using this phrasing, it seams like the value is not valid, but we allow it anyways, but this is not true.

a better phrasing will be:

`The field's value is valid but does not fit the standard format, continue if you know what you are doing`